src/LinearSolverTypes.f90

```diff
@@ -1578,22 +1577,20 @@ SUBROUTINE getResidualNorm_LinearSolverType_Iterative(solver,resid,nIters)
   REQUIRE(ASSOCIATED(solver%x))
   REQUIRE(solver%x%isInit)
 
-  IF(solver%TPLType == PETSC) THEN
+  !IF(solver%TPLType == PETSC) THEN
 #ifdef FUTILITY_HAVE_PETSC
-    IF (PRESENT(nIters)) CALL KSPGetIterationNumber(solver%ksp,niters,ierr)
-    CALL KSPGetResidualNorm(solver%ksp,resid,ierr)
+    !IF (PRESENT(nIters)) CALL KSPGetIterationNumber(solver%ksp,niters,ierr)
+    !CALL KSPGetResidualNorm(solver%ksp,resid,ierr)
```

Comment:
What's the story with this change?

Reply:
It was a mistake. I'm preparing the merge request for the MPACT side of things and this leaked in from there. I'm fixing it and a few other minor mistakes at the moment.